dbt_project.yml

```diff
@@ -1,6 +1,6 @@
 config-version: 2
 name: 'netsuite'
-version: '0.8.0'
+version: '0.8.1'
```

Comment:
Bump index to 0.9.0. Also be sure to bump the version in the README

Reply:
updatec